reference/versions/0.5.0/reference.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
So `./reference` should be for camshaft's dependants and `./lib` should introspect to discover its own analysis, right?